package.json

```diff
@@ -35,6 +35,7 @@
     "css-loader": "^6.7.3",
```

Comment:
Could you drop the `package-lock.json` and `package.json` changes here? I see another PR of yours changes these and it'll likely conflict.

(On another note - perhaps we should add `package-lock.json` to `.gitignore` and remove it from the repo entirely).

Reply:
According to [stackoverflow](https://stackoverflow.com/questions/48524417/should-the-package-lock-json-file-be-added-to-gitignore), we shouldn't add package-lock.json to gitignore, instead we should use `npm ci` to install dependencies instead of `npm i`

Reply:
Yes, I think we should allow package-lock updates only if there is a dependency change